server/models/db/availability.js

```diff
@@ -0,0 +1,38 @@
+const { db } = require('../../lib/connection');
+
+//const getAvailabilityDetails = require('../data/AvailabilityDetails');
+
+const Tables = require('../../constants/tables.json');
+
+//Parameters returned are slot_id, school_id, worker_id, status, and date
+async function getAvailabilityDetails( workerId, schoolId, startTime, endTime) {
+    try{
+    //var queryParams = `${Tables.Worker_Timeslot}.slot_id, ${Tables.workerTimeslot}.school_id, ${Tables.timeslot}.worker_id, ${Tables.timeslot}.date, ${Tables.Worker_Timeslot}.status`;
```

Comment:
Hi Nancy, 
As previously mentioned, for commented out code lines, I would suggest adding a brief explanation at the top OR removing the lines altogether to prevent future confusion and unnecessarily long files.

Reply:
Hi Amy,
Thank you for suggesting that!

My action: I've removed the suggested comments. The code is now concise and to the point.

You can now see these changes in line 8-11 of server/models/db/availability.js

Please confirm if this takes care of your suggestion!